### PR TITLE
Fixed basePath

### PR DESCRIPTION
**Pull request by gorco**
(Now the basePath variable is needed, see and add it in config-values file)

### CODE REVIEW

**Review by RotaruDan — COMMENTED** [review]
As we discussed, it's a good practice to play with the X-Forwarded-* headers from the proxy as to dinamically change the basePath (and using the req.config.a2.a2Prefix) 

More info about the headers: 
https://plone.lucidsolutions.co.nz/web/reverseproxyandcache/setting-nginx-http-x-forward-headers-for-reverse-proxy

File: app/viewRoutes.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ var express = require('express'),
     router = express.Router();
 
 router.get('/', function (req, res) {
-    res.render('login');
+    res.render('login', {basePath: req.app.config.basePath});
```
Comment (RotaruDan) [inline app/viewRoutes.js]: We realised that this is needed for each route (/signup, /class...)

File: bin/viewRoutes.js [hunk 2]
```diff
@@ -21,6 +21,10 @@
 var express = require('express'),
     router = express.Router();
 
+router.get('/login', function (req,res) {
+    res.render('login');
```
Comment (RotaruDan) [inline bin/viewRoutes.js]: I think this file is never used and can be deleted, could you confirm please

**Comment by gorco** [comment]
Yep, I removed the file

**Pull request merged**